drake/core/Path.h.in

```diff
@@ -2,7 +2,7 @@
 
 #include <string>
 
-namespace Drake {
+namespace {
```

Comment:
Google style says "Do not use unnamed namespaces in .h files." (it can break the linker).  This seems like a header file, and thus this is a defect?  Also, the change to line 5 turns the closing-brace comment (argh!) in line 10 to be lies.